tests/installation/partitioning_togglehome.pm

```diff
@@ -35,7 +35,7 @@ sub run {
         elsif (is_storage_ng && !get_var('LVM')) {
             $cmd{toggle_home} = 'alt-o';
         }
-        send_key $cmd{toggle_home};
+        wait_screen_change { send_key $cmd{toggle_home} };
```

Comment:
Because this key stroke was missed. With respect that we are dealing with ncurses pop up and that there is no video available, I guess the issue is again that we are pressing keystrokes while yast is loading default data.